src/dotnet/Chat/Markup/Internal/ParserExt.cs

```diff
@@ -11,23 +11,23 @@ internal static class ParserExt
 
     public static Action<string>? DebugOutput { get; set; }
 
-    public static Parser<char, T> Debug<T>(this Parser<char, T> parser, Func<T, string> formatter)
+    public static Parser<char, T> Debug<T>(this Parser<char, T> parser, Func<T, string> formatter) where T: Markup
```

Comment:
I would rather remove "where" or put `[Conditional("DEBUG")]` here.

Reply:
Another alternative is to use `where T : class` - i.e. no reason to limit this just to `Markup`.